pinot-core/src/test/java/org/apache/pinot/queries/ForwardIndexDisabledSingleValueQueriesTest.java

```diff
@@ -845,6 +845,69 @@ public void testSelectQueriesWithReload()
       assertEquals(resultRow[0], 240528);
       assertEquals(resultRow[1], "gFuH");
     }
+
+    // Re-enable forward index for column9, column11, and column6
```

Comment:
Suggest doing this for both SV and MV columns.

Reply:
done. one issue here, the MV column test is taking 7-8 minutes to run :( it passes but maybe this is too long. Any suggestions to improve this?

Also couldn't run `SELECT *` because some other columns still have forward index disabled in these tests. So aded a test for selecting the specific column instead.

Reply:
The reload isn't taking 7-8 minutes. The verification is taking that long. Need to walk through 400K rows, construct the sets for before and after and validate that they match. So there shouldn't be a major performance issue with the reload part itself.